src/VirtualClient/VirtualClient.Actions/FIO/FioExecutor.cs

```diff
@@ -313,37 +315,15 @@ protected override async Task ExecuteAsync(EventContext telemetryContext, Cancel
                             ErrorReason.WorkloadUnexpectedAnomaly);
                     }
 
-                    IEnumerable<Disk> disksToTest = this.GetDisksToTest(disks);
-
-                    if (disksToTest?.Any() != true)
-                    {
-                        throw new WorkloadException(
-                            "Expected disks to test not found. Given the parameters defined for the profile action/step or those passed " +
-                            "in on the command line, the requisite disks do not exist on the system or could not be identified based on the properties " +
-                            "of the existing disks.",
-                            ErrorReason.DependencyNotFound);
-                    }
-
-                    if (await this.CreateMountPointsAsync(disksToTest, cancellationToken).ConfigureAwait(false))
```

Comment:
In FIO we still need to create mount points,

Reply:
I changed the profile to reflect that